seed/serializers/properties.py

```diff
@@ -276,12 +293,22 @@ def to_representation(self, data):
             iterable = data
             view_ids = [view.id for view in iterable]
             results = []
+
+            # If data is provided, grab extra_data columns to be shown in the result
+            if iterable:
```

Comment:
Ah, yes, this works too. Just grab the columns on the first iteration... well done.